pkg/jobrunaggregator/cmd.go

```diff
@@ -36,8 +34,6 @@ func NewJobAggregatorCommand() *cobra.Command {
 	cmd.AddCommand(releasebigqueryloader.NewBigQueryReleaseTableCreateFlagsCommand())
 	cmd.AddCommand(releasebigqueryloader.NewBigQueryReleaseUploadFlagsCommand())
 
-	cmd.AddCommand(tablescreator.NewBigQueryCreateTablesFlagsCommand())
```

Comment:
Poking around I found a reference to this in a [README](https://github.com/openshift/ci-tools/blob/2e31ca598ffe40a935f941e23e4e57f15aa0f316/cmd/job-run-aggregator/README.md?plain=1#L37) probably a lot else out of date there so this is a minor nit.